src/Administration/Migration/V6_7/Migration1737472122TokenUser.php

```diff
@@ -0,0 +1,39 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Administration\Migration\V6_7;
+
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Migration\MigrationStep;
+
+/**
+ * @internal
+ */
+#[Package('after-sales')]
+class Migration1737472122TokenUser extends MigrationStep
+{
+    public function getCreationTimestamp(): int
+    {
+        return 1737472122;
+    }
+
+    public function update(Connection $connection): void
```

Comment:
As your migration is only in 6.7 namespace I assume this feature will land only in 6.7?

Then please adjust based on the oauth lib update we wanna do, basically all interfaces changed because of native types, see https://github.com/shopware/shopware/pull/5986

Reply:
I have create a ticket to check that: 

https://shopware.atlassian.net/browse/NEXT-40597

and i do not resolve this conversation